wcivf/apps/elections/templates/elections/includes/_elections_breadcrumbs.html

```diff
@@ -0,0 +1,11 @@
+<nav class="ds-breadcrumbs ds-stack" aria-label="You are here: {{ request.path }}">
+    <ol>
+      <li>
+        <a href="{% url 'home_view' %}">Home</a>
+      </li>
+      <li>
+        <a href="{% url 'elections_view' %}">Elections</a>
+      </li>
```

Comment:
I've noticed quite a lot of repetition in the breadcrumbs for the elections. To save on this, you could add a block here to allow extra crumbs to be added e.g.:
```
<nav class="ds-breadcrumbs ds-stack" aria-label="You are here: {{ request.path }}">
    <ol>
        <li>
            <a href="{% url 'home_view' %}">Home</a>
        </li>
        <li>
            <a href="{% url 'elections_view' %}">Elections</a>
        </li>
        {% block extra_crumbs %}
        {% endblock %}
    </ol>
</nav>
```
Then in the other breadcrumb includes in the election app, you can extend from this file so that you always have the "Home" and "Elections" crumbs, and then you only need to add the extra crumbs to include. So for example, in the `_fptp_breadcrumbs.html` file you would have:
```
{% extends "elections/includes/_elections_breadcrumbs.html" %}

{% block extra_crumbs %}
    <li>
        First-past-the-post Voting System
    </li>
{% endblock extra_crumbs %}
```
Then repeat this process for the other breadcrumb include files in the app. That way ensures you always have the base elections breadcrumbs without having to repeat them over and over (and stops odd ones being missed out).